src/main/java/graphql/execution/ValuesResolver.java

```diff
@@ -146,8 +146,7 @@
     public Map<String, Object> getArgumentValues(List<GraphQLArgument> argumentTypes,
                                                  List<Argument> arguments,
                                                  Map<String, Object> coercedVariables) {
-        GraphQLCodeRegistry codeRegistry = GraphQLCodeRegistry.newCodeRegistry().fieldVisibility(DEFAULT_FIELD_VISIBILITY).build();
-        return getArgumentValuesImpl(codeRegistry, argumentTypes, arguments, coercedVariables);
+        return getArgumentValuesImpl(DEFAULT_FIELD_VISIBILITY, argumentTypes, arguments, coercedVariables);
     }
```

Comment:
We only took the GraphQLCodeRegistry so we could get the field visibility

So we collapsed it so we no longer instantiate an object to get to static